mediation_service/mediation/prepare_ssp_response.py

```diff
@@ -154,7 +168,49 @@ def __transform_allergy_local_references(ssp_response: dict):
                 ] = patient_list.get(patient_id[1], patient_ref)
 
 
-def __filter_non_allergy_intolerance(ssp_response: dict):
+def _extract_resolved_allergies(ssp_response: dict):
+    """
+    Select resolved allergy intolerance resources from Ended allergies list
```

Comment:
Do you guys add parameter and return value explanations to docstrings? Not sure what the convention is but it can help to improve function readability.